src/main/java/seedu/address/model/person/Tutorial.java

```diff
@@ -0,0 +1,68 @@
+package seedu.address.model.person;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.util.AppUtil.checkArgument;
+
+/**
+ * Represents a Student's tutorial in StudentConnect.
+ * Guarantees: immutable;
+ */
+public class Tutorial {
+
+    public static final String MESSAGE_CONSTRAINTS = "Tutorials should be integers between 01 and 22, "
+                                                        + "separated by a comma.";
+
```

Comment:
insert this line of code
Constraint: Tutorial should only contain numbers, and it should be 2 digit long between 01 and 22
```suggestion
public static final String VALIDATION_REGEX = "^(0[1-9]|1\d|2[0-2])$";
```